packages/cli/src/run-test.ts

```diff
@@ -8,9 +8,25 @@ import * as query from './query';
 import { StreamingFormatter } from './format-helpers';
 
 export function* runTest(config: ProjectOptions, formatter: StreamingFormatter): Operation<void> {
-  let client: Client = yield Client.create(`ws://localhost:${config.port}`);
+
+  let uri = `ws://localhost:${config.port}`;
+  
+  let client: Client = yield function*() {
+    try {
+      return yield Client.create(uri);
+    } catch (e) {
+      if (e.message.includes('websocket server closed connection unexpectedly')) {
+        throw new MainError({ 
+          exitCode: 1,
+          message: `Could not connect to BigTest server on ${uri}. Run "bigtest server" to start the server.`
+        });
+      }
```

Comment:
Can we raise a more specific error in the client in case this happens, so we don't have to rely on matching on the error message?

Reply:
Done

Reply:
Oops... nevermind, not done yet.

Reply:
Ok, done now